listen.js

```diff
@@ -0,0 +1,7 @@
+const app = require("./app");
```

Comment:
This isn't needed for this project. We will eventually host this project online so won't need to make requests through insomnia/localhost, and for now the listener created for us when we use request(app) will suffice, as we are driving our development through our testing.